packages/web-components/fast-foundation/src/listbox/listbox.spec.ts

```diff
@@ -0,0 +1,153 @@
+import { expect } from "chai";
+import { customElement, DOM, html } from "@microsoft/fast-element";
+import { OptionTemplate as itemTemplate, Option } from "../option";
+import { fixture } from "../fixture";
+import { Listbox, ListboxTemplate as template } from "./index";
+
+@customElement({
+    name: "fast-list-box",
```

Comment:
I think we'll want this as a single word, `listbox`.

Reply:
big oof